daemon/containerd/service.go

```diff
@@ -78,10 +78,9 @@ func (i *ImageService) Cleanup() error {
 	return nil
 }
 
-// GraphDriverName returns the name of the graph drvier
-// moved from Daemon.GraphDriverName, used by:
-// - newContainer
-func (i *ImageService) GraphDriverName() string {
+// StorageDriver returns the name of the default storage-driver (snapshotter)
+// used by the ImageService.
+func (i *ImageService) StorageDriver() string {
 	return ""
```

Comment:
currently always empty, but will be propagated once https://github.com/rumpl/moby/pull/40 is upstreamed